src/app.js

```diff
@@ -1,5 +1,191 @@
+/* eslint-disable prefer-const */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable consistent-return */
```

Comment:
All these should be removed, eslint exceptions are for just that, exceptions to the rule. None of these are needed here